inc/class-job.php

```diff
@@ -12,6 +12,7 @@ class Job {
 	public $start;
 	public $nextrun;
 	public $interval;
+	public $schedule;
```

Comment:
This should probably default to `__fake_schedule` or whatever we called it?

Reply:
Done in 38c423c97be93477f817cae4b750862a876da0f7